### PR TITLE
Wait Time

### PR DESCRIPTION
**Pull request by jjennifergu**
## Overview

Refactor schema to align with backend changes for wait times.

## Changes Made

- Remove force unwrapping for error description to prevent crashes
- Update `Schema` and `SchemaToModel` to factor in new changes on the backend for eatery wait times
- Should display wait times for current day for each eatery

## Test Coverage

Playtesting

## Screenshots

simulator_screenshot_7D043786-D6E4-4C74-9271-607753B6BD57

### CODE REVIEW

**Review by antoinztte — APPROVED** [review]
LGTM

**Review by vinnie4k — APPROVED** [review]
Looks good!

File: Eatery Blue/Model/EateryFormatter.swift [hunk 1]
```diff
@@ -222,8 +222,10 @@ class EateryFormatter {
 
             if minutesLow > walkTimeMinutesCap + waitTimeMinutesCap {
                 return ">\(walkTimeMinutesCap + waitTimeMinutesCap) min"
-            } else {
+            } else if minutesLow < minutesHigh {
                 return "\(minutesLow)-\(minutesHigh) min"
+            } else {
```
Comment (ReadeP) [inline Eatery Blue/Model/EateryFormatter.swift:227]: This can be a one liner

Comment (ReadeP) [reply Eatery Blue/Model/EateryFormatter.swift:227]: Use the ternary operator

**Review by ReadeP — APPROVED** [review]

**Pull request merged**